README.md

```diff
@@ -3,29 +3,28 @@
 
 # Serverless Workflow Specification - Java SDK
 
-Provides the Java API/SPI and Model Validation for the [Serverless Workflow Specification](https://github.com/serverlessworkflow/specification)
+Provides the Java API for the [Serverless Workflow Specification](https://github.com/serverlessworkflow/specification)
 
 With the SDK you can:
 
-* Parse workflow JSON and YAML definitions
-* Programmatically build workflow definitions
-* Validate workflow definitions (both schema and workflow integrity validation)
-* Generate workflow diagram (SVG)
-* Set of utilities to help runtimes interpret the Serverless Workflow object model
+* Read workflow JSON and YAML definitions
+* Write workflow in JSON and YAML format. 
 
-Serverless Workflow Java SDK is **not** a workflow runtime implementation but can be used by Java runtime implementations
-to parse and validate workflow definitions as well as generate the workflow diagram (SVG).
+Serverless Workflow Java SDK is **not** a workflow runtime implementation but can be used by Java runtime implementations to parse workflow definitions.
 
 ### Status
 
 | Latest Releases | Conformance to spec version |
 | :---: | :---: |
+| [7.0.0.Final](https://github.com/serverlessworkflow/sdk-java/releases/tag/7.0.0.Final) | [v0.10](https://github.com/serverlessworkflow/specification/tree/0.10.x) |
 | [5.0.0.Final](https://github.com/serverlessworkflow/sdk-java/releases/tag/5.0.0.Final) | [v0.8](https://github.com/serverlessworkflow/specification/tree/0.8.x) |
 | [4.0.5.Final](https://github.com/serverlessworkflow/sdk-java/releases/tag/4.0.5.Final) | [v0.8](https://github.com/serverlessworkflow/specification/tree/0.8.x) |
 | [3.0.0.Final](https://github.com/serverlessworkflow/sdk-java/releases/tag/3.0.0.Final) | [v0.7](https://github.com/serverlessworkflow/specification/tree/0.7.x) |
 | [2.0.0.Final](https://github.com/serverlessworkflow/sdk-java/releases/tag/2.0.0.Final) | [v0.6](https://github.com/serverlessworkflow/specification/tree/0.6.x) |
 | [1.0.3.Final](https://github.com/serverlessworkflow/sdk-java/releases/tag/1.0.3.Final) | [v0.5](https://github.com/serverlessworkflow/specification/tree/0.5.x) |
 
+Note that 6.0.0.Final, which will be the one for specification version 0.9, is skipped intentionally in case someone want to work on it. 
```

Comment:
```suggestion
Note that `6.0.0.Final`, which will be the one for specification version 0.9, is skipped intentionally in case the community expresses intentions to work on it. 
```